### DISCUSSION TITLE
Blocked account

### DISCUSSION THREAD

**Discussion by kbklub** [vercel/community#3847] [Account Issue] [answered]
### Question

I tried creating a vercel account, but I got "This user account is blocked." when trying to login with github. Please can anyone tell me how to resolve this?

**Comment by Cozacs** [comment]
i`m having the same problem as u, so if u discover a form to use vercel tell me then.

**Reply by phetotmartinez** [reply]
me either, same problem

**Reply by Cozacs** [reply]
Ok so the problem with my account was solved by sending a e-mail to registration@vercel.com I recommend to everyone try to solve with this method

**Comment by oezeb** [comment]
same issue here

I wrote on 29/08/2023 to registration@vercel.com about the problem and I got a case number. Two working days has passed and since then I didn't hear back from them.

**Comment by nialexsan** [comment]
I have the same issue, and I'm still waiting for this issue to be resolved
the initial ticket had been submitted on Aug 28

**Comment by bthaile** [comment]
Same problem, my account is blocked. Are they blocking crypto organizations?

**Comment by itzmiftah** [comment]
same problem my account is blocked

**Comment by Developer0000000** [comment]
same issue😭😭

**Comment by ElliotReed** [comment]
same

**Comment by j0mba** [comment]
same issue

**Comment by cw-roy** [comment]
Same issue here as well.

**Reply by cw-roy** [reply]
It took a few days but I finally received a response after sending an email to registration@vercel.com. They were able to unlock the account. My specific issue is resolved.

**Reply by Cozacs** [reply]
Ok so the problem with my account was solved by sending a e-mail to registration@vercel.com I recommend to everyone try to solve with this method

**Comment by RoboScriptSmith** [comment]
My versal account has been blocked please unblock it.

**Comment by zonarolo** [comment]
I have a colleague who has had the same problem for almost a month, he has written a couple of times to github thinking that this was his problem when trying to login with github from vercel but they have replied from github saying that they are not the ones with the problem and to contact vercel.

He has done so a couple of times to the registration email mentioned in this thread but still no response. 

If any of you get a solution, please let us know.

**Comment by njorogewambuielizabeth** [comment]
Same problem my account is blocked

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with blocked accounts. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)
